teuthology/orchestra/remote.py

```diff
@@ -217,6 +217,17 @@ def move_file(self, src, dst, sudo=False, mode=None, owner=None,
             args += ' && ' + chown + ' ' + owner + ' ' + dst
         self.run(args=args)
 
+    def is_mounted(self, path):
+        """
+        Check if the given path is mounted on the remote machine.
+
+        This method checks the contents of "/proc/self/mounts" instead of
+        using "mount" or "findmnt" command since these commands hang when a
+        CephFS client is blocked and it's mount point on the remote machine
+        is left unhandled/unmounted.
+        """
+        return path in self.read_file('/proc/self/mounts', stdout=StringIO())
```

Comment:
I think the idea of this is fine, but the implementation isn't quite there. As is this will return `True` if `path` is e.g. `"ext4"`, `""`, `"\n"`, etc...

Reply:
Also: Why `/proc/self/mounts/` as opposed to `/proc/mounts`?

Reply:
Fixed.